script/e2e-test.sh

```diff
@@ -229,6 +255,11 @@ if [[ -n "${TEST_UPGRADE}" ]]; then
 
   info "Waiting for Kubeapps components to be ready..."
   k8s_wait_for_deployment kubeapps kubeapps-ci
+
+  # TODO(agamez): Remove these lines in the next version
+  kubectl delete  clusterrole -n kubapps kubeapps:controller:kubeops-ns-discovery-kubeapps kubeapps:controller:kubeops-operators-kubeapps kubeapps:kubeapps:apprepositories-read kubeapps:kubeapps:apprepositories-refresh kubeapps:kubeapps:apprepositories-write || true
+  kubectl delete  clusterrolebinding -n kubapps kubeapps:controller:kubeapps:apprepositories-read kubeapps:controller:kubeops-ns-discovery-kubeapps || true
+  kubectl delete apprepositories.kubeapps.com -n kubeapps bitnami || true
```

Comment:
I am not sure you should need these?

The clusterrole/clusterrolebinding should be adopted by the new version and the apprepository should not exist because we are using `--set apprepository.initialRepos=null`

Reply:
Unfortunately I think you saw a former commit, 7086d6f6d578c43d5c13b50400804d40a71da1f9 should fix it.
Thanks for the heads-up anyway :)